third_party/ibis/ibis_teradata/client.py

```diff
@@ -66,7 +66,7 @@ class TeradataClient(SQLClient):
     table_class = TeradataTable
     dialect = compiler.TeradataDialect
 
-    def __init__(self, host, user_name, password, port=1025, use_no_lock_tables=False):
+    def __init__(self, host, user_name, password, port=1025, logmech="TD2", use_no_lock_tables=False):
```

Comment:
I'm not sure if default arguments can be written in double quotes. Pretty sure it has to be `logmech='TD2'`. Have you tested supplying this parameter and making sure it works?